src/PendingCalls/TestCall.php

```diff
@@ -615,6 +622,20 @@ public function coversNothing(): self
         return $this;
     }
 
+    /**
+     * Adds a reference to the tested method or class. This helps to link test
+     * cases to the source code for easier navigation.
```

Comment:
```suggestion
     * Adds one or more references to the tested method or class. This helps
     * to link test cases to the source code for easier navigation.
```

Reply:
I removed the "property" since it’s no longer needed. thanks!